tiny_skia/src/backend.rs

```diff
@@ -154,7 +154,26 @@ impl Backend {
                 border_width,
                 border_color,
             } => {
-                let physical_bounds = (*bounds + translation) * scale_factor;
+                // Tiny Skia's `PathBuilder::finish` returns `None` on infinites and NaNs which causes iced to panic.
+                // Certain widget combinations can lead to an infinite in either dimension hence the need for this check.
```

Comment:
What kind of _valid_ widget combinations could lead to an infinite quad? It's important we don't try to fix invalid use cases.

Reply:
That's a good point! I mainly tried to fix the panic as reported in #2066, but the code is likely wrong since the submitter is sticking a `TextInput` into a `Scrollable` which is what directly leads to the panic. It also doesn't do what they _want_ it do.

With that said, the WGPU renderer doesn't panic whereas the Tiny Skia renderer panics when it unwraps `None`. The message confused the submitter as mentioned in their original issue since it doesn't say anything about what caused the panic.

Would it make sense to merge this so neither renderer panics? Or should the Tiny Skia renderer still panic for the incorrect code?